srcpkgs/wicd/template

```diff
@@ -1,35 +1,40 @@
 # Template file for 'wicd'
 pkgname=wicd
 version=1.7.4
-revision=4
+revision=5
 archs=noarch
-pycompile_module="wicd"
-pycompile_dirs="usr/lib/wicd usr/share/wicd"
 hostmakedepends="python python-setuptools python-Babel"
-depends="dbus-glib dhcpcd ethtool net-tools python-dbus python-gobject2 python-urwid
- util-linux wireless_tools wpa_supplicant"
+depends="dbus-glib dhclient ethtool net-tools python-dbus python-gobject2 python-urwid
+ util-linux wireless_tools wpa_supplicant kdesu"
```

Comment:
`kdesu` will pull a lot of kde dependency, I don't think people want to see that.
I don't use `wicd`, maybe I'm wrong.

Reply:
`wicd` needs one of gksu, kdesu, ktsuss for priviledge elevation. I added kdesu and configured it in manager-settings.conf because it is the only one remaining in the repo. I can remove it and add it as a recommendation in the INSTALL.msg. It is a big dependency...

Reply:
@sgn : Ι moved the kdesu dependency to the wicd-gtk subpackage. It is only needed for the gui and somebody can use wicd-cli or wicd-curses without it.

Reply:
still weird for a gtk package to pull in lots of kde libs.

Reply:
We do not have one of the supported sudo gui frontends in the repo (they are abandoned). Ι can remove it and add a recommendation to the INSTALL.msg if you think that is a better solution.

Reply:
Does wicd-gtk work if you launch it with sudo? If it doesn't, I think it makes sense to have a graphical sudo client as a dependency.

Reply:
It does. A gui sudo helper is an optional dependency - I added it because there is no way to suggest an optional dep except in INSTALL.msg and I am under the impression that is frowned upon...

Reply:
Pushed the changes :)